scss/v2/components/tile.scss

```diff
@@ -83,6 +121,32 @@
     box-shadow: 0 2px 0 0 rgba(65, 64, 66, 0.35);
   }
 
+  &__highlighted-content-image-container {
+    // Remove margin from the image container so that the image can stretch across the full width of the tile.
+    margin-left: -16px;
+    margin-right: -16px;
+  }
+
+  &__highlighted-content-image {
+    width: 100%;
+    height: auto;
+
+    @include breakpoint(sm) {
+      max-height: 180px;
+    }
+  }
+
+  &__highlighted-content-summary {
+    font-size: 18px;
```

Comment:
I was under the impression that would be the case, but it appeared to be reverting it to `14px` as per the `p` tag instead of inheriting from the `tile` class in the parent element. Think I will change this to `inherit`, though